src/main/kotlin/org/rucca/cheese/model/PatchTaskMembership200ResponseDTO.kt

```
@@ -0,0 +1,23 @@
+data class PatchTaskMembership200ResponseDTO(
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("code", required = true)
+    val code: kotlin.Int,
+    @field:Valid
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("data", required = true)
+    val `data`: PatchTaskMembership200ResponseDataDTO,
+    @Schema(example = "null", required = true, description = "")
+    @get:JsonProperty("message", required = true)
+    val message: kotlin.String
+) {}
```

detekt

[warning] The class or object PatchTaskMembership200ResponseDTO is empty.

(detekt.empty-blocks.EmptyClassBlock)